src/doc/en/installation/index.rst

```diff
@@ -4,219 +4,143 @@
-    line prompt, don't type it!) The second step will ask a few questions,
-    and you may need to hit :kbd:`Enter` to confirm or type ``yes``
-    and then hit :kbd:`Enter`.
-
-    .. code-block:: shell
-
-       $ curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
-       $ bash Miniforge3-$(uname)-$(uname -m).sh
-       $ conda create -n sage sage python=3.11
-
-    (If there are any installation failures, please report them to
-    the conda-forge maintainers by opening a `GitHub Issue for
-    conda-forge/sage-feedstock <https://github.com/conda-forge/sage-feedstock/issues>`_.)
-
-    You can now start SageMath as follows:
-
-    .. code-block:: shell
-
-       $ conda activate sage
-       $ sage
-
-    This way of starting Sage gives you the most basic way of using
-    Sage in the terminal. See :ref:`sec-launching` for recommended next steps,
-    in particular for setting up the Jupyter notebook, which is required if
-    you want to use graphics.
+**I don't want to do Sage development:** Then follow instructions in
+:ref:`sec-installation-conda` and :ref:`sec-installation-conda-binary`. This is
+the easiest way of using Sage in the terminal. Then see :ref:`sec-post-installation` for
+recommended next steps, in particular for setting up the Jupyter notebook,
+which is convenient if you want to use graphics.
 
 .. _installation-guide-linux:
 
 Linux
-=====
+-----
 
-- **Do you want to do SageMath development?**
+**I want to do Sage development:**
 
-  - **Yes, development:**
+- Obtain the Sage sources via ``git`` as described in `the Sage Developer Guide
+  <https://doc.sagemath.org/html/en/developer/walkthrough.html#chapter-walkthrough>`_.
 
-    Obtain the SageMath sources via ``git`` as described in `The Sage
-    Developer's Guide
-    <https://doc.sagemath.org/html/en/developer/walkthrough.html#chapter-walkthrough>`_.
+- Then build Sage from source, choosing ``develop`` git branch, as described in
+  section :ref:`sec-installation-from-sources`.
 
-    - Then build SageMath from source as described in section
-      :ref:`sec-installation-from-sources`.
+- Alternatively, follow the instructions in section
+  :ref:`sec-installation-conda-develop`;
+  these describe an experimental method that gets all required
+  packages, including Python packages, from conda-forge.
 
-    - Alternatively, follow the instructions in section
-      :ref:`sec-installation-conda-develop`;
-      these describe an experimental method that gets all required
-      packages, including Python packages, from conda-forge.
+**I don't want to do Sage development:**
 
-  - No development: **Do you have root access (sudo)?**
+- **I have root access (can use sudo command):** Then the easiest way to install Sage is
+  through a Linux distribution that provides it as a package.  Some
+  Linux distributions have up-to-date versions of Sage,
```

Comment:
I think this can be improved. Instead of sending people to repology, just name the very few distros (arch, gentoo) that actually have an up-to date version of sage. And given that this is not the case for the bigger players like ubuntu, I would put the conda install first again (as it will apply to most people).

Reply:
Sending people to repology seems better because it would have up to date information. If we hard-code them, it would be easily out of date.

I added
```diff
   You may need to consider upgrading your Linux distribution because a recent
   version of Sage is only available on a newer version of the distribution.
 
+  If there is no up-to-date version for your Linux distribution, you may
+  consider `conda`, as explained below.
+
```
instead of recommending conda first.

Reply:
> Sending people to repology seems better because it would have up to date information. If we hard-code them, it would be easily out of date.

Maybe do both? `If you are on Gentoo or Arch Linux (or [any other distribution that provides an up-to-date Sage package](<link to repology>), then use your system package manager to install Sage. Otherwise ...`. This is definitely more user-friendly, which should be the major aim of these instructions.

Reply:
How can we be sure that Gentoo and Arch Linux will provide an up-to-date sage? Is Gentoo providing an up-to-date sage now? It is not in the current repology list.

Reply:
> How can we be sure that Gentoo and Arch Linux will provide an up-to-date sage? 

If it's outdated, we need to update the recommendation. Same as for https://doc.sagemath.org/html/en/installation/linux.html.

Reply:
As said in the PR description, one of the goals is to make the instructions in the readme as stable as possible so that we could only focus on the main installation guide as update occasions arise.